drivers/sensor/iis3dhhc/iis3dhhc.c

```diff
@@ -37,15 +37,15 @@ static int iis3dhhc_sample_fetch(struct device *dev,
 }
 
 #define IIS3DHHC_FROM_LSB_TO_ums2(lsb)	\
-		((IIS3DHHC_FROM_LSB_TO_mg((s64_t)lsb) * SENSOR_G) / 1000LL)
+		((iis3dhhc_from_lsb_to_mg(lsb) * SENSOR_G) / 1000LL)
 
 static inline void iis3dhhc_convert(struct sensor_value *val,
 					s16_t raw_val)
 {
 	s64_t micro_ms2;
 
 	/* Convert to m/s^2 */
-	micro_ms2 = IIS3DHHC_FROM_LSB_TO_ums2((s64_t)raw_val);
+	micro_ms2 = IIS3DHHC_FROM_LSB_TO_ums2(raw_val);
```

Comment:
Is that normal this function kept its format ?
Previous iis3dhhc_from_lsb_to_mg was put in whole minor case..

Reply:
Yes, because iis3dhhc_from_lsb_to_mg() is part of stmemsc API, while IIS3DHHC_FROM_LSB_TO_ums2() is defined by me.

Reply:
Then, should it be aligned or made clear that one is from the lib and the other one zephyr custom ?

Reply:
So, what is your proposal?
Actually I think that I can remove it because it is used only in the conversion routine, where I can directly do the calculation. What do you think?

Reply:
Sure, fine with me.

Reply:
ok, done.
I also changed the commit message a bit, as I felt it was not clear enough.